BEP188.md

```diff
@@ -0,0 +1,24 @@
+# BEP-?: Early Broadcast block for an in-turn validator.
+
+## 1. Summary 
+
+This BEP introduces a new block broadcast strategy to improve the network capacity and stability. In-turn validators can broadcast its block earlier when it is ready, so that other nodes could import the block earlier and have more time to mine or import the next block.
```

Comment:
The sentence could be removed from Summary to keep it short.
`In-turn validators can broadcast its block earlier when it is ready, so that other nodes could import the block earlier and have more time to mine or import the next block.`
And `This BEP introduces a new block broadcast strategy to improve the network capacity and stability.  ` to `This BEP introduces a new block broadcast strategy for validators to improve the network capacity and stability.  `